src/it/projects/profile-with-deps-import/pom.xml

```diff
@@ -0,0 +1,42 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.codehaus.mojo.flatten.its</groupId>
+    <artifactId>profile-with-deps-import</artifactId>
+    <version>0.1-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <build>
+        <plugins>
```

Comment:
Maven style is two spaces for indents in XML files, 4 spaces in Java. I'm not sure if that applies to this non-apache plugin though.

Reply:
Agree - should be 2 spaces

Reply:
Fixed